scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -167,12 +182,22 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                                 <li ng-if="type === 'monitoring'">
                                     <button ng-click="aggregate.edit('reorder', true)">
                                         <i class="icon-switches"></i>
-                                        &nbsp;<span>{{:: 'Rearrange Groups' | translate}}</span>
+                                        &nbsp;<span>{{ gettext('Rearrange Groups') }}</span>
                                     </button>
                                 </li>
                             </ul>
                         </div>
-            
+                    </div>
+                    <!-- Active filter tags -->
+                    <div class="search-tags" ng-if="aggregate.activeFilterTags.length > 0">
+                        <ul class="search-tags__list">
+                            <li ng-repeat="tag in aggregate.activeFilterTags">
+                                <div class="tag-label tag-label--highlight2">
+                                    <span>{{ gettext('Cp:') }}</span>{{ tag.label }}<button class="tag-label__remove" ng-click="aggregate.removeFilter(tag.key)"><i class="icon-close-small"></i></button>
```

Comment:
I'd avoid abbreviations and instead use one full label for the [entire group](https://user-images.githubusercontent.com/2076953/72244220-16d88a80-35ee-11ea-82db-176813c00fee.png)